Sources/SingleFactorAuth/SingleFactorAuth.swift

```diff
@@ -29,7 +29,7 @@ public class SingleFactorAuth {
         return .init(privateKey: privKey, publicAddress: publicAddress)
     }
 
-    public func getKey(loginParams: LoginParams) async throws -> TorusSFAKey {
+    public func getTorusKey(loginParams: LoginParams) async throws -> TorusKey {
```

Comment:
Instead of doing this, I think we should keep it as a non-breaking change and only make the inline changes for the dependency within this function.

Reply:
This is non breaking changes
the function signature of `getKey` is still the same

Reply:
I see.